spec-0007/index.md

```diff
@@ -97,44 +97,69 @@ The [deprecation strategy](https://github.com/scientific-python/specs/pull/180#i
 
    ```
 
-   Also note the suggested language for the `rng` parameter docstring.
+   Also note the suggested language for the `rng` parameter docstring, which encourages the user to pass a `Generator` or None, but allows for other types accepted by `numpy.random.default_rng`.
 
 ### Impact
 
-The following users will be affected:
-
-1. Those who use `np.random.seed`.
-   The proposal will do away with that global seeding mechanism, meaning that code that relies on it will, after a certain deprecation period, start seeing a different stream of random numbers than before.
-   To ensure that this does not go unnoticed, the library should raise a `FutureWarning` if `np.random.seed` was called earlier (we show how to do that further down).
+There are three classes of users, which will be affected to varying degrees.
 
-   Code will, in effect, go from being seeded to being unseeded. To avoid that from happening, users should switch from using `np.random.seed` to specifying the `rng` argument to all functions that allow it.
+1. Those who do not attempt to control the random state.
+   Their code will immediately switch from using the unseeded global `RandomState` to using an unseeded `Generator`.
```

Comment:
Note I've reordered these in order of least to greatest impact.
Previously this one said "Their code will, after the deprecation period, use the newly proposed default." I don't think that's true, given the behavior fo `check_random_state`. They *immediately* begin to use `default_rng` instead of the unseeded global `RandomState`. Right?